README.md

```diff
@@ -25,7 +25,7 @@ and passing in data from your content's metadata.
 
 The metadata is just Elm data that you define
 however you want, using a Json Decoder to grab
-data out of your markdown frontmatter.
+data out of your markdown formatter.
```

Comment:
markdown frontmatter is actually a thing. It looks like this:

```md
---
title: YAML Front Matter
description: A very simple way to add structured data to a page.
---
# Here is

the _actual_ markdown
```

It’s not in the markdown “standard”, it’s a thing invented by static site generators.